mne/time_frequency/tests/test_tfr.py

```diff
@@ -715,7 +715,7 @@ def test_add_channels():
     tfr_badsf = tfr_eeg.copy()
     with tfr_badsf.info._unlock():
         tfr_badsf.info['sfreq'] = 3.1415927
-    tfr_eeg = tfr_eeg.crop(-.1, .1)
+    tfr_eeg = tfr_eeg.crop(.1, .1)
```

Comment:
cropping to exactly one sample? why this change?

Reply:
It warned that `-0.1` wasn't in the time range now, which I just wanted to avoid writing a warning catch

Reply:
ok, it's fine to avoid catching the warning, but the test should test something *sensible*: cropping to a single sample is rarely done.

Reply:
The times are `[0.1, 0.2, 0.3]` and tmax is `0.1` so there's no other options... it's not a very realistic test